android/assets/jsons/translations/template.properties

```diff
@@ -789,6 +789,13 @@ Transportation upkeep =
 Unit upkeep = 
 Trades = 
 Units = 
+Unit Supply = 
+Base Supply = 
+Total Supply = 
+In Use = 
+Supply Deficit = 
+Production Penalty = 
+Increase your supply or reduce the amount of units to remove the production malus = 
```

Comment:
use "penalty" instead of "malus", it's a nice word but not in line with our goal of being as simple to understand as possible :)